src/app/Web3Provider.tsx

```diff
@@ -25,8 +25,8 @@ const metadata = {
 };
 const projectId = process.env.NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID!;
 const alchemyId = process.env.NEXT_PUBLIC_ALCHEMY_ID!;
-
-const { contracts } = Tenant.current();
+const { contracts, ui } = Tenant.current();
+const shouldHideAgoraBranding = ui.hideAgoraBranding;
```

Comment:
for me this makes sense because we don't even need to add a default -- if the tenant does not have this flag set, it will be undefined, which will be false, so it's default false. Then, if you want to hide, you have to go through the work of setting a flag. 

happy to change though if the other way around makes sense to majority of other folks